google-cloud-language/lib/google/cloud/language/document.rb

```diff
@@ -180,7 +180,8 @@ def language= new_language
         #   Optional. The default is `false`. If every feature option is
         #   `false`, **all** features will be performed.
         # @param [String] encoding The encoding type used by the API to
```

Comment:
The format and language arguments have the param type of `[String]`, but accept symbols as well. I'll update their docs as well.